app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,551 @@
+    <string name="torrent_rename_success">已成功重命名 Torrent 名称。</string>
+    <string name="torrent_location_update_success">已成功更改 Torrent 的保存位置。</string>
+    <string name="torrent_rename_name_cannot_be_blank">Torrent 名称不得为空。</string>
+    <string name="torrent_location_cannot_be_blank">路径不得为空。</string>
+    <string name="torrent_dlspeed_limit_change_success">已成功更改下载速度限制。</string>
+    <string name="torrent_upspeed_limit_change_success">已成功更改上传速度限制。</string>
+
+    <string name="torrent_share_limit_global">使用全局分享限制</string>
+    <string name="torrent_share_limit_disable">禁用分享限制</string>
+    <string name="torrent_share_limit_custom">设置自定义分享限制</string>
+    <string name="torrent_share_limit_hint_ratio">分享率</string>
+    <string name="torrent_share_limit_hint_time">做种时间（分钟）</string>
+
+    <string name="torrent_no_categories">未找到任何分类。</string>
+    <string name="torrent_no_tags">未找到任何标签。</string>
+
+    <string name="torrent_overview_progress">%1$s / %2$s • %3$s%%</string>
+    <string name="torrent_overview_information">信息</string>
+    <string name="torrent_overview_total_size">总大小</string>
+    <string name="torrent_overview_added_on">添加于</string>
+    <string name="torrent_overview_hash_v1">哈希值 v1</string>
+    <string name="torrent_overview_hash_v2">哈希值 v2</string>
+    <string name="torrent_overview_save_path">保存路径</string>
+    <string name="torrent_overview_comment">注释</string>
+    <string name="torrent_overview_pieces">区块</string>
+    <string name="torrent_overview_pieces_format">%1$d x %2$s （已完成 %3$d）</string>
+    <string name="torrent_overview_completed_on">完成于</string>
+    <string name="torrent_overview_created_by">创建</string>
+    <string name="torrent_overview_created_on">创建于</string>
+
+    <string name="torrent_overview_transfer">传输</string>
+    <string name="torrent_overview_time_active">活动时间</string>
+    <string name="torrent_overview_downloaded">已下载</string>
+    <string name="torrent_overview_downloaded_format">%1$s （本次会话 %2$s）</string>
+    <string name="torrent_overview_uploaded">已上传</string>
+    <string name="torrent_overview_uploaded_format">@string/torrent_overview_downloaded_format</string>
+    <string name="torrent_overview_ratio">分享率</string>
+    <string name="torrent_overview_reannounce_in">下次汇报</string>
+    <string name="torrent_overview_last_activity">最近活动</string>
+    <string name="torrent_overview_last_seen_complete">最后完整可见</string>
+    <string name="torrent_overview_connections">连接</string>
+    <string name="torrent_overview_connections_format">%1$d （最大%2$d）</string>
+    <string name="torrent_overview_seeds">种子</string>
+    <string name="torrent_overview_seeds_format">%1$d （总计 %2$d）</string>
+    <string name="torrent_overview_peers">用户</string>
+    <string name="torrent_overview_peers_format">@string/torrent_overview_seeds_format</string>
+    <string name="torrent_overview_wasted">已丢弃</string>
+
+    <string name="torrent_files_directory_format"><![CDATA[<b>Directory: </b>]]> %1$s</string>
```

Comment:
I think you forgot to translate `Directory` in this one.

Reply:
Oh, i think it's rss feed code(xml) at first.

Reply:
updated